playwright/network.py

```diff
@@ -97,25 +99,40 @@ async def abort(self, errorCode: str = None) -> None:
 
     async def fulfill(
         self,
-        status: int = None,
+        status: int = 200,
         headers: Dict[str, str] = None,
         body: Union[str, bytes] = None,
+        path: Union[str, Path] = None,
         contentType: str = None,
     ) -> None:
         params = locals_to_params(locals())
-        if contentType:
-            if headers is None:
-                headers = {}
-            headers["Content-Type"] = contentType
-            del params["contentType"]
-        if headers:
-            params["headers"] = serialize_headers(headers)
+        length = 0
         if isinstance(body, str):
             params["body"] = body
             params["isBase64"] = False
+            length = len(body)
```

Comment:
This does not look right - body will be utf-8 encoded, so the length should be byte-length of the encoded string. I wonder if we want to do that on backend (or even in the browser) instead.